src/NServiceBus.Persistence.DynamoDB.Tests/Outbox/OutboxPersisterTests.cs

```diff
@@ -21,6 +21,9 @@ public void SetUp()
         persister = new OutboxPersister(client, new OutboxPersistenceConfiguration(), "endpointIdentifier");
     }
 
+    [TearDown]
```

Comment:
This is a little weird because NUnit now tracks disposables and warns if they are not properly disposed in teardowns. So I had the choice by suppressing the warning around the field declaration vs just implementing the dispose method and I went with the later for now.